# Indonesian translation for apport
# Copyright (c) 2006 Rosetta Contributors and Canonical Ltd 2006
# This file is distributed under the same license as the apport package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2015-09-10 11:16+0200\n"
"PO-Revision-Date: 2015-04-27 21:46+0000\n"
"Last-Translator: Cecep Mahbub <cecep.mahbub@gmail.com>\n"
"Language-Team: Indonesian <id@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2015-12-05 05:09+0000\n"
"X-Generator: Launchpad (build 17862)\n"
"Language: id\n"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:1
#: ../debian/tmp/usr/share/apport/apport-kde.py:468
#: ../debian/tmp/usr/share/apport/apport-kde.py:503
#: ../debian/tmp/usr/share/apport/apport-kde.py:522 ../gtk/apport-gtk.ui.h:1
#: ../kde/apport-kde.py:468 ../kde/apport-kde.py:503 ../kde/apport-kde.py:522
msgid "Apport"
msgstr "Apport"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:2 ../gtk/apport-gtk.ui.h:2
msgid "Cancel"
msgstr "Batal"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:3 ../gtk/apport-gtk.ui.h:3
msgid "OK"
msgstr "OK"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:4 ../gtk/apport-gtk.ui.h:4
msgid "Crash report"
msgstr "Laporan crash"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:5 ../gtk/apport-gtk.ui.h:5
msgid "<big><b>Sorry, an internal error happened.</b></big>"
msgstr "<big><b>Maaf, terjadi kesalahan internal.</b></big>"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:6
#: ../debian/tmp/usr/share/apport/apport-kde.py:240 ../gtk/apport-gtk.ui.h:6
#: ../gtk/apport-gtk.py:300 ../debian/tmp/usr/share/apport/apport-gtk.py:300
#: ../kde/apport-kde.py:240
msgid "If you notice further problems, try restarting the computer."
msgstr ""
"Bila Anda menemui masalah lebih lanjut, cobalah memulai ulang komputer."

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:7 ../gtk/apport-gtk.ui.h:7
msgid "Send an error report to help fix this problem"
msgstr "Mengirim laporan kesalahan dapat membantu memperbaiki masalah ini"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:8 ../gtk/apport-gtk.ui.h:8
msgid "Ignore future problems of this program version"
msgstr "Abaikan masalah atas versi program ini di masa mendatang"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:9
#: ../debian/tmp/usr/share/apport/apport-kde.py:291 ../gtk/apport-gtk.ui.h:9
#: ../gtk/apport-gtk.py:204 ../gtk/apport-gtk.py:573
#: ../debian/tmp/usr/share/apport/apport-gtk.py:204
#: ../debian/tmp/usr/share/apport/apport-gtk.py:573 ../kde/apport-kde.py:291
msgid "Show Details"
msgstr "Tunjukkan Rincian"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:10 ../gtk/apport-gtk.ui.h:10
msgid "_Examine locally"
msgstr "P_eriksa secara lokal"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:11
#: ../debian/tmp/usr/share/apport/apport-kde.py:233 ../gtk/apport-gtk.ui.h:11
#: ../gtk/apport-gtk.py:287 ../debian/tmp/usr/share/apport/apport-gtk.py:287
#: ../kde/apport-kde.py:233
msgid "Leave Closed"
msgstr "Biarkan Tertutup"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:12
#: ../debian/tmp/usr/share/apport/apport-kde.py:230
#: ../debian/tmp/usr/share/apport/apport-kde.py:243 ../gtk/apport-gtk.ui.h:12
#: ../gtk/apport-gtk.py:216 ../gtk/apport-gtk.py:284 ../gtk/apport-gtk.py:303
#: ../debian/tmp/usr/share/apport/apport-gtk.py:216
#: ../debian/tmp/usr/share/apport/apport-gtk.py:284
#: ../debian/tmp/usr/share/apport/apport-gtk.py:303 ../kde/apport-kde.py:230
#: ../kde/apport-kde.py:243
msgid "Continue"
msgstr "Lanjutkan"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:13 ../gtk/apport-gtk.ui.h:13
msgid "<big><b>Collecting problem information</b></big>"
msgstr "<big><b>Mengumpulkan informasi permasalahan</b></big>"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:14 ../gtk/apport-gtk.ui.h:14
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr ""
"Informasi yang dikumpulkan akan dapat membantu para pengembang untuk "
"memperbaiki masalah yang Anda laporkan."

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:15
#: ../debian/tmp/usr/share/apport/apport-kde.py:434 ../gtk/apport-gtk.ui.h:15
#: ../debian/tmp/usr/bin/apport-cli.py:251 ../bin/apport-cli.py:251
#: ../kde/apport-kde.py:434
msgid "Uploading problem information"
msgstr "Mengunggah informasi masalah"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:16 ../gtk/apport-gtk.ui.h:16
msgid "<big><b>Uploading problem information</b></big>"
msgstr "<big><b>Unggah informasi masalah</b></big>"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:17
#: ../debian/tmp/usr/share/apport/apport-kde.py:435 ../gtk/apport-gtk.ui.h:17
#: ../kde/apport-kde.py:435
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr ""
"Iniformasi yang dikumpulan telah dikirim ke sistem pelacak bug. Hal ini "
"membutuhkan waktu beberapa menit."

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr "Berkas crash Apport"

#: ../debian/tmp/usr/share/apport/apport-kde.py:362 ../gtk/apport-gtk.py:142
#: ../debian/tmp/usr/share/apport/apport-gtk.py:142
#: ../debian/tmp/usr/bin/apport-cli.py:150 ../bin/apport-cli.py:150
#: ../kde/apport-kde.py:362
msgid "(binary data)"
msgstr "(data biner)"

#: ../gtk/apport-gtk.py:157 ../debian/tmp/usr/share/apport/apport-gtk.py:157
#, python-format
msgid "Sorry, the application %s has stopped unexpectedly."
msgstr "Maaf, aplikasi %s telah berhenti secara tak terduga."

#: ../gtk/apport-gtk.py:160 ../debian/tmp/usr/share/apport/apport-gtk.py:160
#, python-format
msgid "Sorry, %s has closed unexpectedly."
msgstr "Maaf, %s telah tertutup secara tak terduga."

#: ../debian/tmp/usr/share/apport/apport-kde.py:199
#: ../debian/tmp/usr/share/apport/apport-kde.py:237 ../gtk/apport-gtk.py:165
#: ../debian/tmp/usr/share/apport/apport-gtk.py:165 ../kde/apport-kde.py:199
#: ../kde/apport-kde.py:237
#, python-format
msgid "Sorry, %s has experienced an internal error."
msgstr "Maaf, %s telah mengalami galat internal."

#: ../debian/tmp/usr/share/apport/apport-kde.py:185 ../gtk/apport-gtk.py:177
#: ../debian/tmp/usr/share/apport/apport-gtk.py:177
#: ../debian/tmp/usr/bin/apport-cli.py:178 ../bin/apport-cli.py:178
#: ../kde/apport-kde.py:185
msgid "Send problem report to the developers?"
msgstr "Kirim laporan masalah ke para pengembang?"

#: ../debian/tmp/usr/share/apport/apport-kde.py:193 ../gtk/apport-gtk.py:186
#: ../debian/tmp/usr/share/apport/apport-gtk.py:186 ../kde/apport-kde.py:193
msgid "Send"
msgstr "Kirim"

#: ../gtk/apport-gtk.py:228 ../debian/tmp/usr/share/apport/apport-gtk.py:228
msgid "Force Closed"
msgstr "Paksakan Ditutup"

#: ../debian/tmp/usr/share/apport/apport-kde.py:234
#: ../debian/tmp/usr/share/apport/apport-kde.py:380 ../gtk/apport-gtk.py:229
#: ../gtk/apport-gtk.py:288 ../debian/tmp/usr/share/apport/apport-gtk.py:229
#: ../debian/tmp/usr/share/apport/apport-gtk.py:288 ../kde/apport-kde.py:234
#: ../kde/apport-kde.py:380
msgid "Relaunch"
msgstr "Luncurkan Ulang"

#: ../gtk/apport-gtk.py:238 ../debian/tmp/usr/share/apport/apport-gtk.py:238
#, python-format
msgid "The application %s has stopped responding."
msgstr "Aplikasi %s telah berhenti merespon."

#: ../gtk/apport-gtk.py:242 ../debian/tmp/usr/share/apport/apport-gtk.py:242
#, python-format
msgid "The program \"%s\" has stopped responding."
msgstr "Program \"%s\" telah berhenti merespon."

#: ../debian/tmp/usr/share/apport/apport-kde.py:207 ../gtk/apport-gtk.py:257
#: ../debian/tmp/usr/share/apport/apport-gtk.py:257 ../kde/apport-kde.py:207
#, python-format
msgid "Package: %s"
msgstr "Paket: %s"

#: ../debian/tmp/usr/share/apport/apport-kde.py:213 ../gtk/apport-gtk.py:264
#: ../debian/tmp/usr/share/apport/apport-gtk.py:264 ../kde/apport-kde.py:213
msgid "Sorry, a problem occurred while installing software."
msgstr "Maaf, terjadi masalah ketika memasang perangkat lunak."

#: ../debian/tmp/usr/share/apport/apport-kde.py:219 ../gtk/apport-gtk.py:273
#: ../gtk/apport-gtk.py:292 ../debian/tmp/usr/share/apport/apport-gtk.py:273
#: ../debian/tmp/usr/share/apport/apport-gtk.py:292 ../kde/apport-kde.py:219
#, python-format
msgid "The application %s has experienced an internal error."
msgstr "Aplikasi %s telah mengalami galat internal."

#: ../debian/tmp/usr/share/apport/apport-kde.py:222 ../gtk/apport-gtk.py:276
#: ../debian/tmp/usr/share/apport/apport-gtk.py:276 ../kde/apport-kde.py:222
#, python-format
msgid "The application %s has closed unexpectedly."
msgstr "Aplikasi %s telah tertutup tanpa diharapkan."

#: ../debian/tmp/usr/share/apport/apport-kde.py:244 ../gtk/apport-gtk.py:304
#: ../debian/tmp/usr/share/apport/apport-gtk.py:304 ../kde/apport-kde.py:244
msgid "Ignore future problems of this type"
msgstr "Abaikan masalah tipe ini di masa mendatang"

#: ../debian/tmp/usr/share/apport/apport-kde.py:288 ../gtk/apport-gtk.py:577
#: ../debian/tmp/usr/share/apport/apport-gtk.py:577 ../kde/apport-kde.py:288
msgid "Hide Details"
msgstr "Sembunyikan Rincian"

#: ../bin/apport-unpack.py:22 ../debian/tmp/usr/bin/apport-unpack.py:22
#, python-format
msgid "Usage: %s <report> <target directory>"
msgstr "Cara pakai: %s <laporan> <direktori target>"

#: ../bin/apport-unpack.py:46 ../debian/tmp/usr/bin/apport-unpack.py:46
msgid "Destination directory exists and is not empty."
msgstr "Direktori tujuan ada dan tak kosong."

#: ../debian/tmp/usr/share/apport/apport-kde.py:314 ../kde/apport-kde.py:314
msgid "Username:"
msgstr "Nama pengguna:"

#: ../debian/tmp/usr/share/apport/apport-kde.py:315 ../kde/apport-kde.py:315
msgid "Password:"
msgstr "Sandi:"

#: ../debian/tmp/usr/share/apport/apport-kde.py:405 ../kde/apport-kde.py:405
msgid "Collecting Problem Information"
msgstr "Mengumpulkan Informasi Masalah"

#: ../debian/tmp/usr/share/apport/apport-kde.py:406
#: ../debian/tmp/usr/bin/apport-cli.py:238 ../bin/apport-cli.py:238
#: ../kde/apport-kde.py:406
msgid "Collecting problem information"
msgstr "Mengumpukan informasi masalah"

#: ../debian/tmp/usr/share/apport/apport-kde.py:407 ../kde/apport-kde.py:407
msgid ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."
msgstr ""
"Informasi yang dikumpulkan dapat dikirim ke para pengembang untuk "
"meningkatkan aplikasi. Ini mungkin perlu waktu beberapa menit."

#: ../debian/tmp/usr/share/apport/apport-kde.py:433 ../kde/apport-kde.py:433
msgid "Uploading Problem Information"
msgstr "Mengunggah Informasi Masalah"

#: ../apport/com.ubuntu.apport.policy.in.h:1
msgid "Collect system information"
msgstr "Kumpulkan informasi sistem"

#: ../apport/com.ubuntu.apport.policy.in.h:2
msgid ""
"Authentication is required to collect system information for this problem "
"report"
msgstr ""
"Perlu otentikasi untuk mengumpulkan informasi sistem bagi laporan masalah ini"

#: ../apport/com.ubuntu.apport.policy.in.h:3
msgid "System problem reports"
msgstr "Laporan masalah sistem"

#: ../apport/com.ubuntu.apport.policy.in.h:4
msgid ""
"Please enter your password to access problem reports of system programs"
msgstr ""
"Harap masukkan sandi Anda untuk mengakses laporan masalah dari program sistem"

#: ../kde/apport-kde-mime.desktop.in.h:1 ../kde/apport-kde.desktop.in.h:1
#: ../gtk/apport-gtk.desktop.in.h:1
msgid "Report a problem..."
msgstr "Laporkan masalah..."

#: ../kde/apport-kde-mime.desktop.in.h:2 ../kde/apport-kde.desktop.in.h:2
#: ../gtk/apport-gtk.desktop.in.h:2
msgid "Report a malfunction to the developers"
msgstr "Lapor kegagalan kepada pengembang"

#: ../debian/tmp/usr/bin/apport-valgrind.py:37 ../bin/apport-valgrind.py:37
msgid "See man page for details."
msgstr "Lihat rincian di laman man"

#: ../debian/tmp/usr/bin/apport-valgrind.py:43 ../bin/apport-valgrind.py:43
msgid "specify the log file name produced by valgrind"
msgstr "nyatakan nama berkas log yang dihasilkan oleh valgrind"

#: ../debian/tmp/usr/bin/apport-valgrind.py:46 ../bin/apport-valgrind.py:46
msgid ""
"reuse a previously created sandbox dir (SDIR) or, if it does not exist, "
"create it"
msgstr ""
"pakai ulang dir kotak pasir yang sebelumnya dibuat (SDIR), atau bila belum "
"ada, buatlah"

#: ../debian/tmp/usr/bin/apport-valgrind.py:50 ../bin/apport-valgrind.py:50
msgid ""
"do  not  create  or reuse a sandbox directory for additional debug symbols "
"but rely only on installed debug symbols."
msgstr ""
"jangan buat atau pakai ulang direktori kotak pasir untuk simbol awakutu "
"tambahan tapi hanya bergantung kepada simbol awakutu yang telah dipasang"

#: ../debian/tmp/usr/bin/apport-valgrind.py:54 ../bin/apport-valgrind.py:54
msgid ""
"reuse a previously created cache dir (CDIR) or, if it does not exist, create "
"it"
msgstr ""
"pakai ulang dir singgahan yang sebelumnya dibuat (CDIR), atau bila belum "
"ada, buatlah"

#: ../debian/tmp/usr/bin/apport-valgrind.py:58 ../bin/apport-valgrind.py:58
msgid ""
"report download/install progress when installing packages into sandbox"
msgstr ""
"laporkan kemajuan unduh/pasang ketika memasang paket ke dalam kotak pasir"

#: ../debian/tmp/usr/bin/apport-valgrind.py:62 ../bin/apport-valgrind.py:62
msgid ""
"the executable that is run under valgrind's memcheck tool  for memory leak "
"detection"
msgstr ""
"executable yang dijalankan di bawah perkakas memcheck valgrind untuk deteksi "
"kebocoran memori"

#: ../debian/tmp/usr/bin/apport-valgrind.py:66 ../bin/apport-retrace.py:63
#: ../bin/apport-valgrind.py:66 ../debian/tmp/usr/bin/apport-retrace.py:63
msgid ""
"Install an extra package into the sandbox (can be specified multiple times)"
msgstr ""
"Pasang suatu paket ekstra ke dalam kotak pasir (dapat dinyatakan berkali-"
"kali)"

#: ../debian/tmp/usr/bin/apport-valgrind.py:97 ../bin/apport-valgrind.py:97
#, python-format
msgid "Error: %s is not an executable. Stopping."
msgstr "Galat: %s bukan executable. Menghentikan."

#: ../data/kernel_oops.py:29 ../debian/tmp/usr/share/apport/kernel_oops.py:29
msgid "Your system might become unstable now and might need to be restarted."
msgstr ""
"Sistem Anda mungkin menjadi tak stabil sekarang dan mungkin perlu distart "
"ulang."

#: ../bin/apport-retrace.py:34 ../debian/tmp/usr/bin/apport-retrace.py:34
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr ""
"Jangan letakkan pelacakan baru ke dalam laporan, tapi tulis mereka ke stdout."

#: ../bin/apport-retrace.py:36 ../debian/tmp/usr/bin/apport-retrace.py:36
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr ""
"Mulai sesi gdb interaktif dengan core dump dari laporan (-o diabaikan; tidak "
"menulis laporan)"

#: ../bin/apport-retrace.py:38 ../debian/tmp/usr/bin/apport-retrace.py:38
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr ""
"Tulis laporan yang diubah ke berkas yang diberikan alih-alih mengubah "
"laporan asli"

#: ../bin/apport-retrace.py:41 ../debian/tmp/usr/bin/apport-retrace.py:41
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr "Buang core dump dari laporan setelah regenerasi stack trace"

#: ../bin/apport-retrace.py:43 ../debian/tmp/usr/bin/apport-retrace.py:43
msgid "Override report's CoreFile"
msgstr "Timpa CoreFile laporan"

#: ../bin/apport-retrace.py:45 ../debian/tmp/usr/bin/apport-retrace.py:45
msgid "Override report's ExecutablePath"
msgstr "Timpa ExecutablePath laporan"

#: ../bin/apport-retrace.py:47 ../debian/tmp/usr/bin/apport-retrace.py:47
msgid "Override report's ProcMaps"
msgstr "Timpa ProcMaps laporan"

#: ../bin/apport-retrace.py:49 ../debian/tmp/usr/bin/apport-retrace.py:49
msgid "Rebuild report's Package information"
msgstr "Bangun ulang informasi Package laporan"

#: ../bin/apport-retrace.py:51 ../debian/tmp/usr/bin/apport-retrace.py:51
msgid ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."
msgstr ""
"Bangun suatu kotak pasir sementara dan unduh/pasang paket-paket yang "
"diperlukan dan simbol awakutu di sana; tanpa opsi ini diasumsikan bahwa "
"paket-paket dan simbol awakutu yang diperlukan telah terpasang dalam sistem. "
"Argumen menunjuk ke direktori dasar konfigurasi sistem pemaketan; bila Anda "
"menyatakan \"system\", itu akan memakai berkas konfigurasi sistem, tapi "
"hanya akan dapat melacak ulang crash yang terjadi pada rilis yang kini "
"sedang berjalan."

#: ../bin/apport-retrace.py:53 ../debian/tmp/usr/bin/apport-retrace.py:53
msgid ""
"Report download/install progress when installing packages into sandbox"
msgstr ""
"Laporkan kemajuan pengunduhan/pemasangan ketika memasang paket ke dalam "
"kotak pasir"

#: ../bin/apport-retrace.py:55 ../debian/tmp/usr/bin/apport-retrace.py:55
msgid "Prepend timestamps to log messages, for batch operation"
msgstr "Awali pesan log dengan penanda waktu, bagi operasi batch"

#: ../bin/apport-retrace.py:57 ../debian/tmp/usr/bin/apport-retrace.py:57
msgid ""
"Create and use third-party repositories from origins specified in reports"
msgstr ""

#: ../bin/apport-retrace.py:59 ../debian/tmp/usr/bin/apport-retrace.py:59
msgid "Cache directory for packages downloaded in the sandbox"
msgstr "Direktori singgahan bagi paket yang diunduh dalam kotak pasir"

#: ../bin/apport-retrace.py:61 ../debian/tmp/usr/bin/apport-retrace.py:61
msgid ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."
msgstr ""
"Direktori untuk paket yang dibuka kemasannya. Eksekusi di masa mendatang "
"akan menganggap bahwa paket yang telah diunduh juga diekstrak ke kotak pasir "
"ini."

#: ../bin/apport-retrace.py:65 ../debian/tmp/usr/bin/apport-retrace.py:65
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""
"Path ke berkas dengan informasi otentikasi basis data crash. Ini dipakai "
"ketika menyatakan ID crash untuk mengunggah jejak stack yang dilacak ulang "
"(hanya bila -g, -o, maupun -s tak dinyatakan)"

#: ../bin/apport-retrace.py:67 ../debian/tmp/usr/bin/apport-retrace.py:67
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr ""
"Tampilkan jejak stack yang dilacak ulang dan tanyakan persetujuan sebelum "
"mengirim mereka ke basis data crash."

#: ../bin/apport-retrace.py:69 ../debian/tmp/usr/bin/apport-retrace.py:69
msgid ""
"Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr "Path ke basis data sqlite duplikat (baku: tak ada uji duplikasi)"

#: ../bin/apport-retrace.py:78 ../debian/tmp/usr/bin/apport-retrace.py:78
msgid "You cannot use -C without -S. Stopping."
msgstr "Anda tidak bisa menggunakan -C tanpa -S. Sedang menghentikan."

#. translators: don't translate y/n, apport currently only checks for "y"
#: ../bin/apport-retrace.py:111 ../debian/tmp/usr/bin/apport-retrace.py:111
msgid "OK to send these as attachments? [y/n]"
msgstr "Ok untuk mengirim lampiran-lampiran ini? [y/t]"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:124
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:124
#: ../apport/ui.py:124
msgid "This package does not seem to be installed correctly"
msgstr "Paket ini nampaknya tak terpasang secara benar"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:129
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:129
#: ../apport/ui.py:129
#, python-format
msgid ""
"This is not an official %s package. Please remove any third party package "
"and try again."
msgstr ""
"Ini bukan paket %s resmi. Harap hapus sebarang paket pihak ketiga dan coba "
"lagi."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:146
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:146
#: ../apport/ui.py:146
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""
"Paket yang Anda instal sudah obsolete. Silakan perbaharui paket berikut dan "
"periksa apakah masih terjadi masalah:\n"
"\n"
"%s"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:270
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:270
#: ../apport/ui.py:270
msgid "unknown program"
msgstr "program tak dikenal"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:271
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:271
#: ../apport/ui.py:271
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr "Maaf, program \"%s\" ditutup tak disangka-sangka"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:273
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1312
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:273
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1312
#: ../apport/ui.py:273 ../apport/ui.py:1312
#, python-format
msgid "Problem in %s"
msgstr "Masalah di %s"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:274
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:274
#: ../apport/ui.py:274
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr ""
"Komputer anda sepertinya tidak memiliki cukup memori bebas untuk secara "
"otomatis menganalisa masalah dan mengirimkan laporan pada pengembang."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:322
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:330
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:457
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:460
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:661
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1118
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1284
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1288
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:322
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:330
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:457
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:460
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:661
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1118
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1284
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1288
#: ../apport/ui.py:322 ../apport/ui.py:330 ../apport/ui.py:457
#: ../apport/ui.py:460 ../apport/ui.py:661 ../apport/ui.py:1118
#: ../apport/ui.py:1284 ../apport/ui.py:1288
msgid "Invalid problem report"
msgstr "Laporan masalah tidak valid"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:323
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:323
#: ../apport/ui.py:323
msgid "You are not allowed to access this problem report."
msgstr "Anda tak diijinkan untuk mengakses laporan masalah ini."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:326
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:326
#: ../apport/ui.py:326
msgid "Error"
msgstr "Galat"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:327
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:327
#: ../apport/ui.py:327
msgid "There is not enough disk space available to process this report."
msgstr "Tak tersedia ruang cakram yang cukup untuk mengolah laporan ini."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:411
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:411
#: ../apport/ui.py:411
msgid "No package specified"
msgstr "Tak ada paket yang dinyatakan"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:412
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:412
#: ../apport/ui.py:412
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr ""
"Anda perlu menyatakan suatu paket atau sebuah PID. Lihat --help untuk "
"informasi lebih."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:435
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:435
#: ../apport/ui.py:435
msgid "Permission denied"
msgstr "Tak diijinkan"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:436
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:436
#: ../apport/ui.py:436
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr ""
"Proses berikut tidak dimiliki oleh Anda. Silakan jalankan program ini dengan "
"mengunakan root atau pemilik proses."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:438
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:438
#: ../apport/ui.py:438
msgid "Invalid PID"
msgstr "PID tak valid"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:439
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:439
#: ../apport/ui.py:439
msgid "The specified process ID does not belong to a program."
msgstr "ID proses yang dinyatakan bukan milik suatu program."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:458
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:458
#: ../apport/ui.py:458
#, python-format
msgid "Symptom script %s did not determine an affected package"
msgstr "Skrip gejala %s tak menentukan paket yang terpengaruh"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:461
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:461
#: ../apport/ui.py:461
#, python-format
msgid "Package %s does not exist"
msgstr "Paket %s tidak ada"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:485
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:673
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:678
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:485
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:673
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:678
#: ../apport/ui.py:485 ../apport/ui.py:673 ../apport/ui.py:678
msgid "Cannot create report"
msgstr "Tidak dapat membuat laporan"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:500
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:546
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:563
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:500
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:546
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:563
#: ../apport/ui.py:500 ../apport/ui.py:546 ../apport/ui.py:563
msgid "Updating problem report"
msgstr "Sedang memperbarui laporan masalah"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:501
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:501
#: ../apport/ui.py:501
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""
"Anda bukan pelapor atau pelanggan dari laporan masalah ini, atau laporan "
"adalah duplikat atau sudah ditutup.\n"
"\n"
"Silakan membuat suatu laporan baru memakai \"apport-bug\"."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:510
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:510
#: ../apport/ui.py:510
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""
"Anda bukan pelapor dari laporan masalah ini. Lebih mudah untuk menandai "
"suatu kutu sebagai duplikat dari lainnya daripada memindah komentar dan "
"lampiran Anda ke suatu kutu baru.\n"
"\n"
"Akibatnya, kami menyarankan agar Anda mengirim suatu laporan kutu baru "
"memakai \"apport-bug\" dan membuat komentar dalam kutu ini tentang yang Anda "
"kirimkan.\n"
"\n"
"Apakah Anda yakin ingin meneruskan?"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:547
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:564
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:547
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:564
#: ../apport/ui.py:547 ../apport/ui.py:564
msgid "No additional information collected."
msgstr "Tidak ada informasi tambahan yang dikumpulkan"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:615
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:615
#: ../apport/ui.py:615
msgid "What kind of problem do you want to report?"
msgstr "Masalah macam apa yang ingin Anda laporkan?"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:632
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:632
#: ../apport/ui.py:632
msgid "Unknown symptom"
msgstr "Gejala tak dikenal"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:633
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:633
#: ../apport/ui.py:633
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr "Gejala '%s' tak dikenal."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:664
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:664
#: ../apport/ui.py:664
msgid ""
"After closing this message please click on an application window to report a "
"problem about it."
msgstr ""
"Setelah menutup pesan ini silakan klik pada jendela aplikasi untuk "
"melaporkan masalah tentang hal ini."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:674
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:679
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:674
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:679
#: ../apport/ui.py:674 ../apport/ui.py:679
msgid "xprop failed to determine process ID of the window"
msgstr "xprop gagal untuk menentukan ID proses dari jendela"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:693
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:693
#: ../apport/ui.py:693
msgid "%prog <report number>"
msgstr "%prog <report number>"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:695
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:695
#: ../apport/ui.py:695
msgid "Specify package name."
msgstr "Tentukan nama paket."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:697
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:748
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:697
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:748
#: ../apport/ui.py:697 ../apport/ui.py:748
msgid "Add an extra tag to the report. Can be specified multiple times."
msgstr ""
"Tambahkan label tambahan untuk laporan. Dapat memasukkan beberapa label "
"sekaligus."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:727
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:727
#: ../apport/ui.py:727
msgid ""
"%prog [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr "%prog [opsi] [gejala|pid|paket|path program|berkas .apport/.crash]"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:730
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:730
#: ../apport/ui.py:730
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""
"Mulai dalam moda pelaporan kutu. Memerlukan --package dan opsional --pid, "
"atau hanya --pid. Bila keduanya tak diberikan, tampilkan daftar dari gejala "
"yang telah diketahui. (Otomatis bila argumen tunggal diberikan.)"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:732
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:732
#: ../apport/ui.py:732
msgid "Click a window as a target for filing a problem report."
msgstr "Klik jendela sebagai target untuk mengajukan laporan masalah."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:734
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:734
#: ../apport/ui.py:734
msgid "Start in bug updating mode. Can take an optional --package."
msgstr "Memulai mode memperbarui bug. Dapat mengambil optional --package."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:736
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:736
#: ../apport/ui.py:736
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr ""
"Laporkan kutu tentang suatu gejala. (Otomatis bila nama gejala diberikan "
"sebagai argumen tunggal.)"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:738
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:738
#: ../apport/ui.py:738
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""
"Nyatakan nama paket dalam moda --file-bug. Ini opsional bila suatu --pid "
"dinyatakan.  (Otomatis bila nama paket diberikan sebagai argumen tunggal.)"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:740
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:740
#: ../apport/ui.py:740
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"
msgstr ""
"Nyatakan suatu program yang sedang berjalan dalam mode --file-bug. Bila ini "
"dinyatakan, laporan kutu akan memuat lebih banyak informasi. (diasumsikan "
"bila pid diberikan sebagai satu-satunya argumen.)"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:742
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:742
#: ../apport/ui.py:742
msgid "The provided pid is a hanging application."
msgstr "Pid yang diberikan adalah aplikasi yang hang."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:744
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:744
#: ../apport/ui.py:744
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""
"Laporkan crash dari berkas .apport atau .crash alih-alih dari yang tertunda "
"di %s. (Otomatis bila berkas diberikan sebagai argumen tunggal.)"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:746
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:746
#: ../apport/ui.py:746
msgid ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."
msgstr ""
"Dalam mode pelaporan kutu, simpan informasi yang terkumpul ke dalam suatu "
"berkas alih-alih melaporkannya. Berkas ini kemudian dapat dilaporkan nanti "
"atau dari mesin lain."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:750
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:750
#: ../apport/ui.py:750
msgid "Print the Apport version number."
msgstr "Cetak nomor versi Apport."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:889
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:889
#: ../apport/ui.py:889
msgid ""
"This will launch apport-retrace in a terminal window to examine the crash."
msgstr ""
"Ini akan meluncurkan apport-retrace dalam suatu jendela terminal untuk "
"memeriksa crash."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:890
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:890
#: ../apport/ui.py:890
msgid "Run gdb session"
msgstr "Memulai sesi gdb"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:891
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:891
#: ../apport/ui.py:891
msgid "Run gdb session without downloading debug symbols"
msgstr "Memulai sesi gdb tanpa mengunduh simbol debug"

#. TRANSLATORS: %s contains the crash report file name
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:893
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:893
#: ../apport/ui.py:893
#, python-format
msgid "Update %s with fully symbolic stack trace"
msgstr "Mutakhirkan %s dengan pelacakan stack simbol lengkap"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:969
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:979
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:969
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:979
#: ../apport/ui.py:969 ../apport/ui.py:979
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr ""
"Laporan masalah berlaku bagi suatu program yang telah tak terpasang lagi."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:994
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:994
#: ../apport/ui.py:994
#, python-format
msgid ""
"The problem happened with the program %s which changed since the crash "
"occurred."
msgstr ""
"Terjadi masalah dengan program %s yang berubah semenjak terjadinya crash."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1041
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1075
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1290
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1041
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1075
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1290
#: ../apport/ui.py:1041 ../apport/ui.py:1075 ../apport/ui.py:1290
msgid "This problem report is damaged and cannot be processed."
msgstr "Laporan permasalah gagal dan tidak dapat diproses."

#. package does not exist
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1045
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1045
#: ../apport/ui.py:1045
msgid "The report belongs to a package that is not installed."
msgstr "Laporan ini milik paket yang tidak dipasang."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1049
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1049
#: ../apport/ui.py:1049
msgid "An error occurred while attempting to process this problem report:"
msgstr "Terjadi kesalahan ketika mencoba untuk memproses laporan masalah:"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1119
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1119
#: ../apport/ui.py:1119
msgid "Could not determine the package or source package name."
msgstr "TIdak dapat menentukan nama paket atau nama paket sumber."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1137
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1137
#: ../apport/ui.py:1137
msgid "Unable to start web browser"
msgstr "TIdak dapat menjalankan perambah web"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1138
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1138
#: ../apport/ui.py:1138
#, python-format
msgid "Unable to start web browser to open %s."
msgstr "TIdak dapat menjalankan perambah web untuk membuka %s."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1238
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1238
#: ../apport/ui.py:1238
#, python-format
msgid "Please enter your account information for the %s bug tracking system"
msgstr "Silakan masukkan informasi akun Anda bagi sistem pelacakan kutu %s"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1250
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1250
#: ../apport/ui.py:1250
msgid "Network problem"
msgstr "Masalah di jaringan"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1252
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1252
#: ../apport/ui.py:1252
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr ""
"Tak bisa menyambung ke basis data crash, silakan periksa sambungan Internet "
"Anda."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1279
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1279
#: ../apport/ui.py:1279
msgid "Memory exhaustion"
msgstr "Kehabisan memori"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1280
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1280
#: ../apport/ui.py:1280
msgid "Your system does not have enough memory to process this crash report."
msgstr ""
"Sistem Anda tidak memiliki memori yang cukup untuk memproses laporan crash "
"ini."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1315
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1315
#: ../apport/ui.py:1315
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""
"Permasalahan yang tidak dapat dilaporkan:\n"
"\n"
"%s"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1371
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1378
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1371
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1378
#: ../apport/ui.py:1371 ../apport/ui.py:1378
msgid "Problem already known"
msgstr "Masalah sudah diketahui"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1372
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1372
#: ../apport/ui.py:1372
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""
"Masalah ini telah dilaporkan dalam laporan kutu yang ditampilkan di peramban "
"web. Silakan periksa apakah Anda dapat menambah informasi lebih lanjut yang "
"mungkin berguna bagi para pengembang."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1379
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:1379
#: ../apport/ui.py:1379
msgid "This problem was already reported to developers. Thank you!"
msgstr "Masalah ini sudah dilaporkan ke pengembang. Terima kasih!"

#: ../data/apportcheckresume.py:67
#: ../debian/tmp/usr/share/apport/apportcheckresume.py:67
msgid ""
"This occurred during a previous suspend, and prevented the system from "
"resuming properly."
msgstr ""

#: ../data/apportcheckresume.py:69
#: ../debian/tmp/usr/share/apport/apportcheckresume.py:69
msgid ""
"This occurred during a previous hibernation, and prevented the system from "
"resuming properly."
msgstr ""

#: ../data/apportcheckresume.py:74
#: ../debian/tmp/usr/share/apport/apportcheckresume.py:74
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr ""
"Pemrosesan pelanjutan menggantung sangat dekat dengan ujung dan akan nampak "
"seperti komplit secara normal."

#: ../debian/tmp/usr/bin/apport-cli.py:74 ../bin/apport-cli.py:74
msgid "Press any key to continue..."
msgstr "Tekan sembarang tombol untuk lanjut..."

#: ../debian/tmp/usr/bin/apport-cli.py:81 ../bin/apport-cli.py:81
msgid "What would you like to do? Your options are:"
msgstr "Apa yang ingin Anda lakukan? Pilihan Anda adalah:"

#: ../debian/tmp/usr/bin/apport-cli.py:85 ../bin/apport-cli.py:85
#, python-format
msgid "Please choose (%s):"
msgstr "Silakan pilih (%s):"

#: ../debian/tmp/usr/bin/apport-cli.py:148 ../bin/apport-cli.py:148
#, python-format
msgid "(%i bytes)"
msgstr "(%i bytes)"

#: ../debian/tmp/usr/bin/apport-cli.py:179 ../bin/apport-cli.py:179
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""
"Setelah laporan masalah dikirim, silakan mengisi formulir di peramban\n"
"web yang otomatis dibuka."

#: ../debian/tmp/usr/bin/apport-cli.py:182 ../bin/apport-cli.py:182
#, python-format
msgid "&Send report (%s)"
msgstr "&Kirim laporan (%s)"

#: ../debian/tmp/usr/bin/apport-cli.py:186 ../bin/apport-cli.py:186
msgid "&Examine locally"
msgstr "P&eriksa secara lokal"

#: ../debian/tmp/usr/bin/apport-cli.py:190 ../bin/apport-cli.py:190
msgid "&View report"
msgstr "&Tilik laporan"

#: ../debian/tmp/usr/bin/apport-cli.py:191 ../bin/apport-cli.py:191
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr ""
"Simpan ber&kas laporan untuk pengiriman nanti atau penyalinan ke tempat lain"

#: ../debian/tmp/usr/bin/apport-cli.py:192 ../bin/apport-cli.py:192
msgid "Cancel and &ignore future crashes of this program version"
msgstr "Batalkan dan aba&ikan crash di masa mendatang dari versi program ini"

#: ../debian/tmp/usr/bin/apport-cli.py:194
#: ../debian/tmp/usr/bin/apport-cli.py:271
#: ../debian/tmp/usr/bin/apport-cli.py:303
#: ../debian/tmp/usr/bin/apport-cli.py:324 ../bin/apport-cli.py:194
#: ../bin/apport-cli.py:271 ../bin/apport-cli.py:303 ../bin/apport-cli.py:324
msgid "&Cancel"
msgstr "&Batal"

#: ../debian/tmp/usr/bin/apport-cli.py:222 ../bin/apport-cli.py:222
msgid "Problem report file:"
msgstr "Berkas laporan masalah:"

#: ../debian/tmp/usr/bin/apport-cli.py:228
#: ../debian/tmp/usr/bin/apport-cli.py:233 ../bin/apport-cli.py:228
#: ../bin/apport-cli.py:233
msgid "&Confirm"
msgstr "&Konfirmasi"

#: ../debian/tmp/usr/bin/apport-cli.py:232 ../bin/apport-cli.py:232
#, python-format
msgid "Error: %s"
msgstr "Galat: %s"

#: ../debian/tmp/usr/bin/apport-cli.py:239 ../bin/apport-cli.py:239
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""
"Informasi yang dikumpulkan dapat dikirim ke para pengembang untuk\n"
"memperbaiki aplikasi. Ini mungkin makan waktu beberapa menit."

#: ../debian/tmp/usr/bin/apport-cli.py:252 ../bin/apport-cli.py:252
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""
"Informasi yang dikumpulan sedang dikirim ke sistem pelacak kutu.\n"
"Ini mungkin makan waktu beberapa menit."

#: ../debian/tmp/usr/bin/apport-cli.py:302 ../bin/apport-cli.py:302
msgid "&Done"
msgstr "&Selesai"

#: ../debian/tmp/usr/bin/apport-cli.py:308 ../bin/apport-cli.py:308
msgid "none"
msgstr "tak ada"

#: ../debian/tmp/usr/bin/apport-cli.py:309 ../bin/apport-cli.py:309
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr "Terpilih: %s. Pilihan berganda:"

#: ../debian/tmp/usr/bin/apport-cli.py:325 ../bin/apport-cli.py:325
msgid "Choices:"
msgstr "Pilihan:"

#: ../debian/tmp/usr/bin/apport-cli.py:339 ../bin/apport-cli.py:339
msgid "Path to file (Enter to cancel):"
msgstr "Path ke berkas (Enter untuk membatalkan):"

#: ../debian/tmp/usr/bin/apport-cli.py:345 ../bin/apport-cli.py:345
msgid "File does not exist."
msgstr "Berkas tidak ada."

#: ../debian/tmp/usr/bin/apport-cli.py:347 ../bin/apport-cli.py:347
msgid "This is a directory."
msgstr "Ini bukan direktori."

#: ../debian/tmp/usr/bin/apport-cli.py:353 ../bin/apport-cli.py:353
msgid "To continue, you must visit the following URL:"
msgstr "Untuk melanjutkan, Anda mesti mengunjungi URL berikut:"

#: ../debian/tmp/usr/bin/apport-cli.py:355 ../bin/apport-cli.py:355
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr ""
"Anda dapat meluncurkan sebuah peramban sekarang, atau menyalin URL ini ke "
"dalam peramban pada komputer lain."

#: ../debian/tmp/usr/bin/apport-cli.py:357 ../bin/apport-cli.py:357
msgid "Launch a browser now"
msgstr "Luncurkan sebuah peramban sekarang"

#: ../debian/tmp/usr/bin/apport-cli.py:371 ../bin/apport-cli.py:371
msgid "No pending crash reports. Try --help for more information."
msgstr "Tak ada laporan crash tertunda. Coba --help untuk informasi lebih."

#, no-c-format, python-format
#~ msgid "Complete report (recommended; %s)"
#~ msgstr "Laporan penuh (disarankan; %s)"

#~ msgid ""
#~ "<big><b>Send problem report to the developers?</b></big>\n"
#~ "\n"
#~ "After the problem report has been sent, please fill out the form in the "
#~ "automatically opened web browser."
#~ msgstr ""
#~ "<big><b>Kirim laporan masalah ke pengembang?</b></big>\n"
#~ "\n"
#~ "Setelah laporan masalah telah dikirim, harap isi formulir pada jendela "
#~ "browser yang otomatis terbuka."

#~ msgid ""
#~ "If you were not doing anything confidential (entering passwords or other "
#~ "private information), you can help to improve the application by reporting "
#~ "the problem."
#~ msgstr ""
#~ "Jika Anda tidak melakukan hal-hal yang bersifat rahasia (memasukkan kata "
#~ "sandi atau informasi pribadi lainnya), Anda dapat membantu untuk memperbaiki "
#~ "kinerja aplikasi ini dengan cara melaporkan masalah."

#~ msgid "Application problem"
#~ msgstr "Masalah pada aplikasi"

#~ msgid "_Send Report"
#~ msgstr "_Kirim Laporan"

#~ msgid "Restart _Program"
#~ msgstr "Restart _Program"

#, no-c-format
#~ msgid ""
#~ "<span weight=\"bold\" size=\"larger\">Sorry, the package \"%s\" failed to "
#~ "install or upgrade.</span>"
#~ msgstr ""
#~ "<span weight=\"bold\" size=\"larger\">Maaf, paket \"%s\" gagal diinstal atau "
#~ "diperbaharui.</span>"

#, no-c-format, python-format
#~ msgid "Reduced report (slow Internet connection; %s)"
#~ msgstr "Laporan singkat (koneksi internet lambat; %s)"

#~ msgid "_Report Problem..."
#~ msgstr "_Laporkan masalah..."

#~ msgid "Kernel problem"
#~ msgstr "Masalah di kernel"

#~ msgid ""
#~ "You can help the developers to fix the package by reporting the problem."
#~ msgstr ""
#~ "Anda dapat membantu pengembang untuk memperbaiki paketnya dengan melaporkan "
#~ "masalahnya."

#, python-format
#~ msgid "This is not a genuine %s package"
#~ msgstr "Ini bukan paket %s yang asli"

#~ msgid "Content of the report"
#~ msgstr "Isi dari laporan"

#~ msgid "Package problem"
#~ msgstr "Masalah pada paket"

#~ msgid "&Report Problem..."
#~ msgstr "Lapo&rkan Masalah..."

#, python-format
#~ msgid "The package \"%s\" failed to install or upgrade."
#~ msgstr "Paket \"%s\" gagal dipasang atau ditingkatkan."

#~ msgid "Your system encountered a serious kernel problem."
#~ msgstr "Sistem Anda mengalami masalah kernel serius."

#~ msgid "You can help the developers to fix the problem by reporting it."
#~ msgstr ""
#~ "Anda dapat membantu para pengembang untuk memperbaiki masalah dengan "
#~ "melaporkannya."

#, python-format
#~ msgid "&Send complete report (recommended; %s)"
#~ msgstr "Kirim laporan lengkap (di&sarankan; %s)"

#, python-format
#~ msgid "Send &reduced report (slow Internet connection; %s)"
#~ msgstr "Ki&rim laporan singkat (sambungan Internet lambat; %s)"

#~ msgid ""
#~ "This occured during a previous suspend and prevented it from resuming "
#~ "properly."
#~ msgstr ""
#~ "Ini terjadi selama suspensi sebelumnya dan mencegahnya melanjutkan dengan "
#~ "benar."

#~ msgid ""
#~ "This occured during a previous hibernate and prevented it from resuming "
#~ "properly."
#~ msgstr ""
#~ "Ini terjadi selama hibernasi sebelumnya dan mencegahnya melanjutkan dengan "
#~ "benar."

#~ msgid ""
#~ "This will remove some large items from the report. These are very useful for "
#~ "developers to debug the problem, but might be too big for you to upload if "
#~ "you have a slow internet connection."
#~ msgstr ""
#~ "Ini akan menghapus beberapa item besar dari laporan. Mereka sangat berguna "
#~ "bagi para pengembang untuk mengawakutu masalah, tapi mungkin terlalu besar "
#~ "bagi Anda untuk mengunggah bila Anda memiliki sambungan Internet yang lambat."

#~ msgid "_Ignore future crashes of this program version"
#~ msgstr "Aba_ikan crash di masa mendatang dari versi program ini"

#~ msgid "%prog [options] <apport problem report | crash ID>"
#~ msgstr "%prog [opsi] <laporan masalah apport | ID crash>"

#~ msgid "incorrect number of arguments; use --help for a short help"
#~ msgstr "cacah argumen salah; gunakan --help untuk bantuan singkat"

#~ msgid ""
#~ "you either need to do a local operation (-s, -g, -o) or supply an "
#~ "authentication file (--auth); see --help for a short help"
#~ msgstr ""
#~ "Anda perlu melakukan operasi lokal (-s, -g, -o) atau memberikan berkas "
#~ "otentikasi (--auth); lihat --help untuk bantuan singkat"

#~ msgid ""
#~ "If you were not doing anything confidential (entering passwords or other\n"
#~ "private information), you can help to improve the application by reporting\n"
#~ "the problem."
#~ msgstr ""
#~ "Bila Anda tak lakukan apapun yang rahasia (memasukkan kata sandi atau "
#~ "informasi\n"
#~ "pribadi lainnya), Anda dapat membantu memperbaiki aplikasi dengan "
#~ "melaporkan\n"
#~ "masalah ini."

#, python-format
#~ msgid "%s closed unexpectedly on %s at %s."
#~ msgstr "%s tertutup tak terduga di %s pada %s."

#~ msgid "Your system might become unstablenow and might need to be restarted."
#~ msgstr ""
#~ "Sistem Anda mungkin menjadi tak stabil saat ini dan mungkin perlu dimulai "
#~ "ulang."
